e2e-tests/portal/cypress/e2e/journeys/maker/proceed-to-review/proceed-to-review.spec.js

```diff
@@ -161,25 +124,21 @@ context('Ensure proceed to review button is only visible once facilities are in
     cy.clickSubmitButton();
 
     // Navigate to the deal in question
-    pages.contract.visit(deal);
+    dashboardDeals.visit();
+    cy.clickDashboardDealLink();
 
     // Ensure facility stage is `Incomplete`
-    pages.contract.loansTransactionsTableRows().each((row, index) => {
+    contract.loansTransactionsTableRows().each((row) => {
       const loanId = row.attr('data-cy').split('-')[1];
-      const loan = pages.contract.loansTransactionsTable.row(loanId);
-      const status = index === 2 ? 'Incomplete' : 'Complete';
-      loan.loanStatus().contains(status);
+      const loan = contract.loansTransactionsTable.row(loanId);
+      loan.loanStatus().contains('Incomplete');
     });
   });
 
   it('Ensure proceed to review button is not visible', () => {
-    // Login as a `Maker`
-    cy.login(BANK1_MAKER1);
-
-    // Navigate to the deal in question
-    pages.contract.visit(deal);
+    cy.loginGoToDealPage(BANK1_MAKER1);
 
     // Proceed to review button
-    pages.contract.proceedToReview().should('not.exist');
+    contract.proceedToReview().should('not.exist');
   });
```

Comment:
I dont really follow what this test is checking - it seems to rely on the previous stuff. This is existing though so id leave for now

Reply:
This test checks if Proceed to review button is not visible after clicking on the deal.